read-skipfile-results

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env python3
```

Comment:
add more to the changelog please.

also add an example how to run it.
what you have to run first to get the builds_for_...txt and metadata_list.csv

Reply:
I updated the README to reflect requirements and usage. Have added more to commit message to explain purpose. Let me know if this is appropriate.